.github/workflows/ci.yaml

```
@@ -20,13 +20,12 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        otio-version: ["0.17.0]
-        # TODO Once we have landed extracted adapters we replace otio-version
-        #otio-version: ["0.15.0", "main"]
+        # Use macos-13 so we'll be on intel hardware and can pull a pre-built wheel
+        # When OTIO has an Apple Silicon build we can switch back to macos-latest for that version
+        os: [ubuntu-latest, macos-13, windows-latest]
+        otio-version: ["0.17.0"]
```

```suggestion
        otio-version: ["0.17.0", "main"]
```
We check against main on the other adapters as well